src/x11/x.c

```diff
@@ -584,15 +584,46 @@ static dimension_t x_render_layout(cairo_t *c, colored_layout *cl, colored_layou
         int bg_height = MAX(settings.notification_height, (2 * settings.padding) + h);
         double bg_half_height = settings.notification_height/2.0;
         int pango_offset = (int) floor(h/2.0);
+        double degrees = M_PI / 180.0;
```

Comment:
This should be a `const`, we don't need to modify it.

Reply:
@dj95 He meant to solely mark it with `const` (instead of defining a new macro).